migrations/20231209213904_init-addresses.sql

```diff
@@ -0,0 +1,33 @@
+-- List of supported blockchain namespaces
+CREATE TYPE namespaceses AS ENUM (
+  'eip155' -- Ethereum
+);
+
+-- Initializing the addresses table
+CREATE TABLE addresses (
+  -- Breakdown of the CAP-10 address format into namespace:chain_id:address
+  namespace namespaceses,
```

Comment:
"namespaces" is the plural of "namespace"

```suggestion
  namespace namespaces,
```

Reply:
Fixed! Thanks for catching this.